### PR TITLE
Change pk to row number 4 5

### PR DESCRIPTION
**Pull request by Troemer16**
Primary key is now row number

### CODE REVIEW

File: src/main/java/com/spinningnoodle/communitymanager/datastorage/GoogleSheets.java [hunk 1]
```diff
@@ -150,9 +150,11 @@ public boolean createEntry() {
         List<Map<String, String>> data = new ArrayList<>();
         List<String> attributes = getAttributesAsStrings(values);
 
-        for (int rowNum = 1; rowNum < values.size(); rowNum++) {
+        for (int listNum = 1; listNum < values.size(); listNum++) {
             HashMap<String, String> row = new HashMap<>();
-            List<Object> rawRow = values.get(rowNum);
+            List<Object> rawRow = values.get(listNum);
+            int rowNum = listNum + 1;
+            row.put("primaryKey",rowNum + "");
```
Comment (windust) [inline src/main/java/com/spinningnoodle/communitymanager/datastorage/GoogleSheets.java:157]: why the + ""?, you can just remove it...If you need to cast it, make it (String) rowNum, or better String.valueOf (rowNum)

File: src/main/java/com/spinningnoodle/communitymanager/datastorage/GoogleSheets.java [hunk 2]
```diff
@@ -176,7 +178,7 @@ public boolean createEntry() {
             if (attributeName.getClass().toString().equals("class java.lang.String")) {
```
Comment (windust) [inline src/main/java/com/spinningnoodle/communitymanager/datastorage/GoogleSheets.java:178]: do assignableFrom, or InstanceOf instaed of checking the "Class" https://www.baeldung.com/java-instanceof

File: src/main/java/com/spinningnoodle/communitymanager/datastorage/GoogleSheets.java [hunk 3]
```diff
@@ -283,19 +285,23 @@ public void setStorageID(String storageID) {
     private int getRowNumber(List<List<Object>> values, String primaryKey){
 
         int rowNumber = 1;
-        for(int rowIndex = 0; rowIndex < values.size(); rowIndex++){
-            if(values.get(rowIndex).get(0).toString().equals(primaryKey)){
-                rowNumber = rowIndex+1;
-                break;
-            }
+//        for(int rowIndex = 0; rowIndex < values.size(); rowIndex++){
```
Comment (windust) [inline src/main/java/com/spinningnoodle/communitymanager/datastorage/GoogleSheets.java:288]: you can remove the comments if not being used... unless is for new stuff

**Review by windust — CHANGES_REQUESTED** [review]

File: src/main/java/com/spinningnoodle/communitymanager/datastorage/GoogleSheets.java (hunk 1 above)
Comment (feltonm) [reply src/main/java/com/spinningnoodle/communitymanager/datastorage/GoogleSheets.java:157]: Made change on the implement OAuth branch so we'll have it.

File: src/main/java/com/spinningnoodle/communitymanager/datastorage/GoogleSheets.java (hunk 2 above)
Comment (feltonm) [reply src/main/java/com/spinningnoodle/communitymanager/datastorage/GoogleSheets.java:178]: made change on the implement OAuth branch so we'll have it.

File: src/main/java/com/spinningnoodle/communitymanager/datastorage/GoogleSheets.java (hunk 3 above)
Comment (feltonm) [reply src/main/java/com/spinningnoodle/communitymanager/datastorage/GoogleSheets.java:288]: Removed on the implement OAuth branch. It does risk the row number changing now.

**Pull request merged**